apps/tlon-mobile/ios/Intents/OpenAppIntent.swift

```diff
@@ -0,0 +1,26 @@
+import AppIntents
+
+struct OpenAppIntent: AppIntent {
+  static var title: LocalizedStringResource = "Open channel"
+  static var description = IntentDescription("Opens the app to the specified channel.")
+  static var openAppWhenRun = true
+  
+  @Parameter(title: "Channel ID", description: "The ID of the channel to open the app to.")
+  var channelId: String?
```

Comment:
there's a much better way to expose channels here: you can define a "query" which could let people search the app's store e.g. by channel title. idk if we actually wanna ship this so I didn't go into that.

that said, replacing this param in the future with a searchable channel entity means people's shortcuts will break. idk how you do things in a backwards compatible way (without just being verbose and exposing all old variables).